runtime/include/j9vmls.h

```diff
@@ -45,8 +45,8 @@ extern "C" {
  * The VM local storage function table.
  */
 typedef struct J9VMLSFunctionTable {
-    UDATA  (JNICALL *J9VMLSAllocKeys)(JNIEnv * env, UDATA * pInitCount, ...) ;
-    void  (JNICALL *J9VMLSFreeKeys)(JNIEnv * env, UDATA * pInitCount, ...) ;
+    UDATA  (*J9VMLSAllocKeys)(JNIEnv * env, UDATA * pInitCount, ...) ;
+    void  (*J9VMLSFreeKeys)(JNIEnv * env, UDATA * pInitCount, ...) ;
```

Comment:
Why change only two of the four?

Reply:
Because the other two functions are not vararg, which is acceptable for Clang.
```
    void*  (JNICALL *J9VMLSGet)(JNIEnv * env, void * key) ;
    void*  (JNICALL *J9VMLSSet)(JNIEnv * env, void ** pKey, void * value) ;
```